config_converter/config_mapper/config_mapper.py

```diff
@@ -76,34 +76,39 @@ def extract_root_dirs(config_obj: config_pb2.Directive) -> tuple:
     plugin_prefix_map = {'source': 'in_'}
     dir_name_map = {'source': 'sources'}
     # these dicts can be updated when more plugins are supported
-    for d in config_obj.directives:
-        if d.name not in plugin_prefix_map:
+    for directive in config_obj.directives:
+        if directive.name not in plugin_prefix_map:
             stats['entities_skipped'] += 1
-            stats['attributes_skipped'] += _get_aggregated_num_attributes(d)
+            stats['attributes_skipped'] += _get_aggregated_num_attributes(
+                directive)
+            stats['warning_logs'] += 1
```

Comment:
I think increasing the stats after the logging statement would be more intuitive. I meant:
```
logging.warning(
                'Skip mapping %s due to missing functionality in master agent',
                directive.name)
stats['warning_logs'] += 1
```

It applies to all the log-related stats.

Reply:
Oh you're right it's much more clearer this way - I will change it everywhere!